src/sdk/accountspace_fs.js

```diff
@@ -463,12 +473,52 @@ class AccountSpaceFS {
     // INTERNAL FUNCTIONS //
     ////////////////////////
 
-     _get_account_config_path(name) {
-        return get_config_file_path(this.accounts_dir, name);
+    // this function was copied from namespace_fs and bucketspace_fs
+    // It is a fallback that we use, but might be not accurate
+    _translate_error_codes(err, entity) {
+        if (err.rpc_code) return err;
+        if (err.code === 'ENOENT') err.rpc_code = `NO_SUCH_${entity}`;
+        if (err.code === 'EEXIST') err.rpc_code = `${entity}_ALREADY_EXISTS`;
+        if (err.code === 'EPERM' || err.code === 'EACCES') err.rpc_code = 'UNAUTHORIZED';
+        if (err.code === 'IO_STREAM_ITEM_TIMEOUT') err.rpc_code = 'IO_STREAM_ITEM_TIMEOUT';
+        if (err.code === 'INTERNAL_ERROR') err.rpc_code = 'INTERNAL_ERROR';
+        return err;
+    }
```

Comment:
I deleted this function and moved it to native_fs_utils in PR #8224

Reply:
@alphaprinz did you miss this comment?

Reply:
Probably, I will remove.